<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Img_steg_tool</class>
 <widget class="QMainWindow" name="Img_steg_tool">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Image Steg Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="Tab">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="MainWindow">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>主窗口</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,1">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="ShowImgWidget" native="true"/>
        </item>
        <item>
         <widget class="QWidget" name="SideBar" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QPushButton" name="OpenImg">
               <property name="text">
                <string>打开图片</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="SaveCurrentImg">
               <property name="text">
                <string>保存当前图片</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="CleanCurrentImg">
               <property name="text">
                <string>清除当前图片</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QWidget" name="widget_3" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="BinaryWindow">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>显示二进制</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTextBrowser" name="ShowBinaryBrowser"/>
        </item>
        <item>
         <widget class="QWidget" name="ControlPanel" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="LSBWindow">
       <attribute name="title">
        <string>LSB图像隐写</string>
       </attribute>
       <widget class="QLabel" name="encodelabel_1">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>50</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 85, 255);
font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;隐写文本&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="encodelabel_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>100</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>输入隐写文本：</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="encodetextEdit">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>130</y>
          <width>281</width>
          <height>231</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="encodelabel_3">
        <property name="geometry">
         <rect>
          <x>72</x>
          <y>390</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>输入加密密码：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="encodelineEdit">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>390</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="encodepushButton">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>470</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>隐写并保存</string>
        </property>
       </widget>
       <widget class="QLabel" name="decodelabel_1">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>50</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 85, 255);
font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;提取文本&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="decodepushButton">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>140</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>提取</string>
        </property>
       </widget>
       <widget class="QLabel" name="decodelabel_3">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>230</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>提取隐写文本：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="decodelineEdit">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>100</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="decodetextEdit">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>260</y>
          <width>281</width>
          <height>231</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="decodelabel_2">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>100</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>输入解密密码：</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="encodecheckBox">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>390</y>
          <width>71</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>显示密码</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="decodecheckBox">
        <property name="geometry">
         <rect>
          <x>750</x>
          <y>100</y>
          <width>71</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>显示密码</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="encodeprogressBar">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>430</y>
          <width>241</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QLabel" name="encodelabel_4">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>430</y>
          <width>31</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>进度：</string>
        </property>
       </widget>
       <widget class="QLabel" name="decodelabel_4">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>180</y>
          <width>31</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>进度：</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="decodeprogressBar">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>180</y>
          <width>241</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
